terraform/modules/vpc/endpoints.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_vpc_endpoint" "concourse" {
+  count             = local.zone_count
```

Comment:
I think we just want one endpoint, but attach 3 ENIs to it by providing a list in `subnet_ids` (line 9). `subnet_ids = [aws_subnet.private.*.id]` should do it I think.

Reply:
Actually, come to think of it, I think this PR might be in completely the wrong place? We need an endpoint on the internet egress proxies, not on Concourse!